drivers/sx126x/sx126x.c

```diff
@@ -163,10 +174,17 @@ int sx126x_init(sx126x_t *dev)
         DEBUG("[sx126x] error: no DIO1 pin defined\n");
         return -EIO;
     }
-
+#endif
     /* Reset the device */
     sx126x_reset(dev);
 
+#if IS_USED(MODULE_SX126X_PERIPH)
+    sx126x_set_standby(dev, SX126X_STANDBY_CFG_RC);
+    sx126x_set_dio3_as_tcxo_ctrl(dev, SX126X_TCXO_CTRL_1_7V, 10 << 6 );
+    sx126x_write_register(dev, REG_XTA_TRIM, ( const uint8_t[] ){ 0x00 },1);
+    sx126x_cal(dev, CONFIG_SX126X_CAL_DEFAULT);
```

Comment:
This should also be removed but again, first check that the performance is unaffected.

Reply:
Done. Removed